src/ports/postgres/modules/deep_learning/madlib_keras_fit_multiple_model.sql_in

```diff
@@ -88,6 +88,10 @@ You can set up the models and hyperparameters to try with the
 Model Selection</a> utility to define the unique combinations
 of model architectures, compile and fit parameters.
 
+@note If madlib_keras_fit_multiple_model is running on gpdb5, the database will
+keep on adding to the disk space(depending on the model size) and will only
+release the disk space once the fit multiple query has completed execution.
+
```

Comment:
Suggested slight changes:

@note If 'madlib_keras_fit_multiple_model()' is running on GPDB 5, the database will
keep adding to the disk space (in proportion to model size) and will only
release the disk space once the fit multiple query has completed execution.
This is not the case for GPDB 6+ where disk space is released during the 
fit multiple query.

Reply:
sounds good